packages/core-macro/tests/ifmt.rs

```diff
@@ -1,34 +0,0 @@
-use dioxus_core_macro::*;
```

Comment:
Could we keep testing the behavior of weird formatted expressions with rsx if we don't expose the format macro separately?